config/locales/views/landing_page/de.yml

```diff
@@ -72,36 +72,25 @@ de:
         number_of_pages: "Seitenzahl"
         publisher: "Verleger"
         edition: "Edition"
-        description: "Beschreibung"
-        max_borrowing_days: "Ausleihzeit"
-        max_reservation_days: "Reservierungszeit"
       movie:
-        name: "Name"
         director: "Regisseur"
         release_date: "Erscheinungsdatum"
         format: "Format"
         genre: "Genre"
         language: "Sprache"
         fsk: "Jugendfreigabe"
-        description: "Beschreibung"
-        max_borrowing_days: "Ausleihzeit"
-        max_reservation_days: "Reservierungszeit"
       game:
-        name: "Name"
         author: "Autor"
         illustrator: "Illustrator"
         publisher: "Verleger"
         fsk: "Jugendfreigabe"
         number_of_players: "Spielerzahl"
         playing_time: "Spieldauer"
         language: "Sprache"
-        description: "Beschreibung"
-        max_borrowing_days: "Ausleihzeit"
-        max_reservation_days: "Reservierungszeit"
-    infobox:
-      borrow: 
-        heading: "Wie kann man diesen Artikel ausleihen?"
-        body: "Scannen Sie mit Ihrem mobilen Gerät den QR-Code dieses Artikels. Sie finden den Gegenstand an dem unten angegebenen Ort."
+        infobox:
+          borrow: 
+            heading: "Wie kann man diesen Artikel ausleihen?"
+            body: "Scannen Sie mit Ihrem mobilen Gerät den QR-Code dieses Artikels. Sie finden den Gegenstand an dem unten angegebenen Ort."
```

Comment:
Ich finde die Struktur der Language file jetzt sehr gut! Dann müssen common attribute nur noch einmal angegeben und übersetzt werden 👍🏼 
Hier ist nur eine Kleinigkeit mit der Einrückung schief gegangen. Die Infobox sollte auf der selben Ebene wie `form` liegen. Da die Einrückung nur in der deutschen Datei fehlerhaft ist, laufen alle Tests noch durch, aber wenn man folgenden Code in die `create` Methode im Items Controller packt (um die Reservierungen zu testen).....
```
unless Group.exists?(name: "Default Group")
    @testGroup = Group.new(name: "Default Group")
    @testGroup.save
end

@testGroup = Group.where(name: "Default Group").first
@testMembership = Membership.new(role: 1, user_id: current_user.id, group: @testGroup)
@testMembership.save
user_memberships = current_user.memberships
user_memberships.push(@testMembership)
current_user.save
item_groups = @item.borrower_groups
item_groups.push(@testGroup)
```
...dann sieht man die fehlende Tranlsation auch auf Deutsch:

<img width="511" alt="image" src="https://user-images.githubusercontent.com/65392679/211157735-ec2c0b4d-74f8-4b2f-8769-9bee03b252ab.png">

Reply:
ist jetzt richtig eingerückt :)